src/sentry/api/endpoints/organization_events_stats.py

```diff
@@ -37,6 +37,11 @@ def get(self, request, organization):
                 elif top_events <= 0:
                     return Response({"detail": "If topEvents needs to be at least 1"}, status=400)
 
+            # The partial parameter determins whether or not partial buckets are allowed.
+            # The last bucket of the time series can potentially be a partial bucket when
+            # the start of the bucket does not align with the rollup.
```

Comment:
Hmm more to confirm my own understanding but should this say "when the end of the interval does not align with the rollup"? Like if i had an hourly rollup, and my interval ends at 10:28, we're dropping the last 28 minutes right?

Reply:
It could say start/end because when start does *not* align with the rollup, the end wont either, and vice versa.

But yes, in your example, turning off `partial` means the last 28 minutes are dropped.